packages/kmod-6.12-nvidia-r580/kmod-6.12-nvidia-r580.spec

```diff
@@ -0,0 +1,745 @@
+%{_cross_libdir}/nvidia/tesla/libGLESv2_nvidia.so.2
+%{_cross_libdir}/nvidia/tesla/libGLESv1_CM_nvidia.so.%{tesla_ver}
+%{_cross_libdir}/nvidia/tesla/libGLESv1_CM_nvidia.so.1
+
+# Graphics compat
+%{_cross_libdir}/nvidia/tesla/libEGL.so.1.1.0
+%{_cross_libdir}/nvidia/tesla/libEGL.so.1
+%{_cross_libdir}/nvidia/tesla/libEGL.so.%{tesla_ver}
+%{_cross_libdir}/nvidia/tesla/libGL.so.1.7.0
+%{_cross_libdir}/nvidia/tesla/libGL.so.1
+%{_cross_libdir}/nvidia/tesla/libGLESv1_CM.so.1.2.0
+%{_cross_libdir}/nvidia/tesla/libGLESv1_CM.so.1
+%{_cross_libdir}/nvidia/tesla/libGLESv2.so.2.1.0
+%{_cross_libdir}/nvidia/tesla/libGLESv2.so.2
+
+# NGX
+%{_cross_libdir}/nvidia/tesla/libnvidia-ngx.so.%{tesla_ver}
+%{_cross_libdir}/nvidia/tesla/libnvidia-ngx.so.1
+
+# Firmware
+%{_cross_libdir}/firmware/nvidia/%{tesla_ver}/gsp_ga10x.bin
+%{_cross_libdir}/firmware/nvidia/%{tesla_ver}/gsp_tu10x.bin
+
+# Neither nvidia-peermem nor nvidia-drm are included in driver container images, we exclude them
+# for now, and we will add them if requested
+%exclude %{_cross_datadir}/nvidia/tesla/module-objects.d/nvidia-peermem.mod.o
+%exclude %{_cross_datadir}/nvidia/tesla/module-objects.d/nvidia-peermem.o
+%exclude %{_cross_datadir}/nvidia/tesla/module-objects.d/nvidia-drm.mod.o
+%exclude %{_cross_datadir}/nvidia/tesla/module-objects.d/nvidia-drm.o
+%exclude %{_cross_libexecdir}/nvidia/tesla/bin/nvidia-cuda-mps-control
+%exclude %{_cross_libexecdir}/nvidia/tesla/bin/nvidia-cuda-mps-server
+%exclude %{_cross_bindir}/nvidia-cuda-mps-control
+%exclude %{_cross_bindir}/nvidia-cuda-mps-server
+%if "%{_cross_arch}" == "x86_64"
+%exclude %{_cross_libexecdir}/nvidia/tesla/bin/nvidia-ngx-updater
+%exclude %{_cross_bindir}/nvidia-ngx-updater
+%endif
+
+# None of these libraries are required by libnvidia-container, so they
+# won't be used by a containerized workload
+%exclude %{_cross_libdir}/nvidia/tesla/libGLX.so.0
+%exclude %{_cross_libdir}/nvidia/tesla/libGLdispatch.so.0
+%exclude %{_cross_libdir}/nvidia/tesla/libOpenGL.so.0
+%exclude %{_cross_libdir}/nvidia/tesla/libglxserver_nvidia.so.%{tesla_ver}
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-gtk2.so.%{tesla_ver}
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-gtk3.so.%{tesla_ver}
+%exclude %{_cross_libdir}/nvidia/tesla/nvidia_drv.so
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-egl-wayland.so.1
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-egl-gbm.so.1
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-egl-gbm.so.1.1.2
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-egl-wayland.so.1.1.19
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-egl-xcb.so.1
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-egl-xcb.so.1.0.1
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-egl-xlib.so.1
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-egl-xlib.so.1.0.1
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-sandboxutils.so.1
+%exclude %{_cross_libdir}/nvidia/tesla/libnvidia-sandboxutils.so.%{tesla_ver}
```

Comment:
For context:

I was trying to figure out why `libnvidia-nvvm70.so.4` is new, and I saw it was listed in `sandboxutils-filelist.json`. Apparently, that's the file that lists all the libraries:

```
o A JSON file ('/usr/share/nvidia/files.d/sandboxutils-filelist.json'),
which lists all the driver files (library, firmware, binary,
configuration, ...) used by container runtime environments such as
nvidia-container-toolkit and enroot.
```

That got me thinking "sandboxutils might be important". It turns out it is:

https://github.com/NVIDIA/nvidia-container-toolkit/pull/629

We will have to package it and ship it, as a follow up. Not blocking this change tho. I created an issue to track this:

https://github.com/bottlerocket-os/bottlerocket-kernel-kit/issues/260